backend/src/main/resources/application.yaml

```diff
@@ -30,9 +30,7 @@
 #      timezoneId: "Europe/Brussels"
 #      property-predicates:
 #        fullname: "http://schema.org/name"
-##        capacity: "https://blue-bike.be/ns#capacity"
 #        available: "http://schema.mobivoc.org/#currentValue"
-##        used: "https://w3id.org/gbfs#bikes_in_use"
 #graphdb:
 #  url: "http://localhost:8080/rdf4j-server/repositories/"
 #  repositoryId: "test"
```

Comment:
a lot of comments.. do we still need it ?

Reply:
It's a bad practice in the demonstrator not to remove the application.yaml file, like we do in the server and ldio